_includes/fonts.html

```diff
@@ -1,2 +1,3 @@
 <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
 <link href="https://fonts.googleapis.com/css?family=Bitter:400,400i,700" rel="stylesheet">
+<script src="https://use.fontawesome.com/releases/v5.0.12/js/all.js"></script>
```

Comment:
I think this can be included only when `site.footer_links` exists. Also the script has been [updated](https://fontawesome.com/start):
```suggestion
{% if site.footer_links %}<script defer src="https://use.fontawesome.com/releases/v5.8.2/js/all.js" integrity="sha384-DJ25uNYET2XCl5ZF++U8eNxPWqcKohUUBUpKGlNLMchM7q4Wjg2CUpjHLaL8yYPH" crossorigin="anonymous"></script>{% endif %}
```